scripts/build.sh

```diff
@@ -20,7 +20,7 @@ META_SERVERS_OPTS="-Ddev_meta=$dev_meta -Dfat_meta=$fat_meta -Duat_meta=$uat_met
 
 # =============== Please do not modify the following content =============== #
 # go to script directory
-cd "${0%/*}"
+cd "${0%/*}" || exit 
```

Comment:
what does this change mean?

Reply:
Passing parameters is "unstable" and "unsafe"
When an error occurs, just exit,